client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionDetailsFields/InteractionExternalizationForm/InteractionExternalizationForm.tsx

```diff
@@ -10,17 +10,22 @@ import Toggle from 'commons/Toggle/Toggle';
 import useFormStyles from 'styles/formStyles';
 
 import useStyles from './InteractionExternalizationFormStyles';
+import repetitiveFieldTools from "../../RepetitiveEventForm/hooks/repetitiveFieldTools";
```

Comment:
change to single quotes

Reply:
yeah thanks, fixed all quotes related comments you mentioned [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1301/commits/1e1932acc9a7758c1717647d38df2cdd4eef9dd1)